guide/src/examples/wasm-audio-worklet.md

```diff
@@ -0,0 +1,53 @@
+# WASM audio worklet
+
+[View full source code][code] or [view the compiled example online][online]
+
+[online]: https://wasm-bindgen.netlify.app/exbuild/wasm-audio-worklet/
+[code]: https://github.com/rustwasm/wasm-bindgen/tree/master/examples/wasm-audio-worklet
+
+This is an example of using threads inside specific worklets with WebAssembly,
+Rust, and `wasm-bindgen`, culminating in an oscillator demo. This demo should
+complement the [parallel-raytrace][parallel-raytrace] example by
+demonstrating an alternative approach using ES modules with on-the-fly module
+creation.
+
+[parallel-raytrace]: https://rustwasm.github.io/docs/wasm-bindgen/examples/raytrace.html
+
+### Building the demo
+
+One of the major gotchas with threaded WebAssembly is that Rust does not ship a
+precompiled target (e.g. standard library) which has threading support enabled.
+This means that you'll need to recompile the standard library with the
+appropriate rustc flags, namely
+`-C target-feature=+atomics,+bulk-memory,+mutable-globals`.
+Note that this requires a nightly Rust toolchain. See the [more detailed
+instructions][build] of the parallel-raytrace example.
+
+[build]: https://rustwasm.github.io/docs/wasm-bindgen/examples/raytrace.html#building-the-demo
+
+### Running the demo
+
+With `--target web` you'll be able to use `import` inside of each web worker or
+worklet to import the shim JS generated by `wasm-bindgen` as well as calling
+the `wasm_bindgen` initialization function with the shared memory instance from
+the main thread. The expected usage is that WebAssembly on the main thread will
+post its memory object to all other threads to get instantiated with.
```

Comment:
I'm not sure what you're trying to say in this section. Are you trying to say that `--target web` is required? Or are you describing `dependent_module!`?

Reply:
This section was similar to the corresponding section in raytrace.md. I didn't like it too, so I have removed it.